documentation.md

```diff
@@ -0,0 +1,35 @@
+# delfi_attributes.json - Documentation
+
+This file explains the schema according to wich the `delfi_attributes.json` file is built and should be parsed.
+
+## Schema
+
+The file is a List `[]` of Dictionarys `{}`. 
+
+Every Dictionary must contain the following attributes:
+
+### `DELFI_ID`
+
+An integer that represents the DELFI-ID of the attribute that is in question. 
+
+
+### `LABEL`
+
+A string that represents the Label exactly as it is displayed in the DELFI-Book.
+
+### `CONDITION`
+
+Condition describes what conditions must be met so that the element is tested if they fulfill all conditions stated in `TAGS`. So, to reiterate, `CONDITION` selects the elements for which the tests are performed, while `TAGS` are the conditions that are tested for.
+
+The value can either be a `CONDITION-dicionary` or a list of `CONDITION-dictionary`s. If its a list, the conditions are combined with an OR-operator.
```

Comment:
Please replace last sentence with: "In case of a list only one of the supplied `CONDITION-dicionary`s has to evaluate to true."